x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -251,10 +251,13 @@ export const BenchmarksTable = ({
     return <ErrorMessageComponent error={error} />;
   }
 
+  const benchmarksSorted = benchmarks.sort((benchmarkDataA, benchmarkDataB) =>
+    benchmarkDataA.id.localeCompare(benchmarkDataB.id)
+  );
```

Comment:
We can update this so the sort function does not directly mutate the benchmarks array to avoid unpredictable behaviour. Also, we can wrap it into a memo so it doesn't re-sort the benchmarks array every time the component renders.

```js
const benchmarksSorted = useMemo(() => {
  return [...benchmarks].sort((benchmarkDataA, benchmarkDataB) =>
    benchmarkDataA.id.localeCompare(benchmarkDataB.id)
  );
}, [benchmarks]);
```